test/CMakeLists.txt

```diff
@@ -50,6 +50,11 @@ function(add_umf_test)
 
     set_tests_properties(${TEST_NAME} PROPERTIES LABELS "umf")
 
+    if(WIN32)
+        set_property(TEST ${TEST_NAME} PROPERTY ENVIRONMENT_MODIFICATION 
```

Comment:
Why is this needed? link_libraries should be enough

Reply:
On Win, link_libraries is used to link to .lib with export declarations, but we have to also provide a path to .dll file which is located in a different folder (e.g. build/lib/Release/umf.lib vs build/bin/Release/umf.dll)